doc/src/modules/physics/mechanics/examples/four_bar_linkage_example.rst

```diff
@@ -28,50 +27,57 @@ lengths and density of the links. ::
    >>> q1, q2, q3, u1, u2, u3 = dynamicsymbols('q1:4, u1:4')
    >>> l1, l2, l3, l4, rho = symbols('l1:5, rho')
 
-With all symbols defined, we can now define the bodies. ::
+With all symbols defined, we can now define the bodies and initialize our
+instance of :class:`~.System`. ::
 
    >>> N = ReferenceFrame('N')
-   >>> inertias = [inertia(N, 0, 0, rho * l ** 3 / 12) for l in (l1, l2, l3, l4)]
-   >>> link1 = Body('Link1', frame=N, mass=rho * l1, central_inertia=inertias[0])
-   >>> link2 = Body('Link2', mass=rho * l2, central_inertia=inertias[1])
-   >>> link3 = Body('Link3', mass=rho * l3, central_inertia=inertias[2])
-   >>> link4 = Body('Link4', mass=rho * l4, central_inertia=inertias[3])
+   >>> mass_centers = [Point(f'mc{i}') for i in range(1, 5)]
+   >>> inertias = [Inertia.from_inertia_scalars(P, N, 0, 0, rho*l**3/12)
+   ...             for P, l in zip(mass_centers, (l1, l2, l3, l4))]
+   >>> link1 = RigidBody('Link1', frame=N, mass=rho*l1,
+   ...                   masscenter=mass_centers[0], inertia=inertias[0])
+   >>> link2 = RigidBody('Link2', mass=rho*l2, masscenter=mass_centers[1],
+   ...                   inertia=inertias[1])
+   >>> link3 = RigidBody('Link3', mass=rho*l3, masscenter=mass_centers[2],
+   ...                   inertia=inertias[2])
+   >>> link4 = RigidBody('Link4', mass=rho*l4, masscenter=mass_centers[3],
+   ...                   inertia=inertias[3])
+   >>> system = System.from_newtonian(link1)
 
-Next, we also define the first three joints. ::
+Next, we also define the first three joints creating the open and add them to
```

Comment:
grammar: creating the open {?}

Reply:
Thank you, addressed in f2c5191d8c4a5d77b12111a6d80fcdfb033236ce